src/sources/docker_logs.rs

```diff
@@ -657,6 +669,12 @@ fn add_hostname(mut event: Event, host_key: &str, hostname: &Option<String>) ->
     event
 }
 
+#[derive(Copy, Clone, Debug, Eq, PartialEq)]
+enum Persistence {
```

Comment:
I suggest renaming this to something like `ErrorType`. The name `Persistence` doesn't indicate what is persisting, which led me to think it was about the container at first, not the error from a container.